.github/workflows/large-pr-labeler.yml

```
@@ -2,6 +2,8 @@ name: Label Large PR
 
 on:
   pull_request:
+permissions:
+  pull-requests: write
     types: [opened, synchronize, reopened]
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)